src/fast_gicp/cuda/gaussian_voxelmap.cu

```diff
@@ -224,7 +224,7 @@ void GaussianVoxelMap::create_voxelmap(const thrust::device_vector<Eigen::Vector
 
   thrust::for_each(thrust::cuda::par.on(stream), points.begin(), points.end(), accumulate_points_kernel(voxelmap_info_ptr.data(), buckets, num_points, voxel_means, voxel_covs));
 
-  thrust::for_each(thrust::counting_iterator<int>(0), thrust::counting_iterator<int>(voxelmap_info.num_voxels), ndt_finalize_voxels_kernel(num_points, voxel_means, voxel_covs));
+  thrust::for_each(thrust::cuda::par.on(stream), thrust::counting_iterator<int>(0), thrust::counting_iterator<int>(voxelmap_info.num_voxels), ndt_finalize_voxels_kernel(num_points, voxel_means, voxel_covs));
```

Comment:
no idea why github shows these changes, there is no diff to master that I can see.

Reply:
seems to be result of some rebase-problem.